src/components/formsCommons/WithSwal.js

```diff
@@ -30,7 +30,7 @@ export const withSwal = ({
                 icon: "success",
                 confirmButtonText: confirmButtonText || "OK"
             }).then(swalResult => {
-                if (swalResult.isConfirmed) {
+                if (swalResult.isConfirmed && successFunction) {
```

Comment:
Może głupie pytanie ale czemu musimy sprawdzać czy istnieje successFunction?

Reply:
Bo nie musimy jej zawsze przekazywać. Jeśli na przykład chcesz usunąć ofertę, to wystarczy jedynie wyświetlić powiadomienie.